File: isnull.py
import pandas as pd

df = pd.DataFrame({
    'Nama': [None, 'Niko', 'Gaby', 'Vio', 'Maria'],
    'Umur': [12, 13, None, 18, 34],
    'Kelas': [12, 13, 12, 16, None]
})
print(df)
print()
print(df[df['Umur'].isnull()])
print('\nNot Null')
print(df[df['Umur'].notnull()])
print('\n\n\n\n\n')
print(df.isnull().sum())
print()
print(df.isnull().sum(axis=1))
print()
print(df[df[['Umur', 'Kelas']].isna().any(axis=1)])
print()
print(df[df.isnull().any(axis=1)])
print()
print(df[df.isnull().sum(axis=1) > 0])
print()
print(df.loc[:, df.isnull().any(axis=0)])
print()
print(df.loc[:, df.isnull().sum() > 0])

File: isnull.r
# Load library
library(dplyr)

# Buat DataFrame
df <- data.frame(
    Nama = c(NA, "Niko", "Gaby", "Vio", "Maria"),
    Umur = c(12, 13, NA, 18, 34),
    Kelas = c(12, 13, 12, 16, NA)
)

print(df)
cat("\n")
print(df[is.na(df$Umur), ])
cat("\n")
print(df[!is.na(df$Umur), ])
cat("\n")
print(rowSums(is.na(df)))
cat("\n")
print(colSums(is.na(df)))
cat("\n")
print(df[rowSums(is.na(df[, c("Nama", "Kelas")])) > 0, ])
cat("\n")
print(df[rowSums(is.na(df)) > 0, ])
cat("\n")
print(df[, colSums(is.na(df)) > 0])
